python/adelphi/integration-tests/adelphi-docker-tests.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
```

Comment:
This is the main shell script for the integration tests, it starts up the Docker containers for each C*, populates the base schema, runs the anonymizer and compares the output with the expected result.

This is mostly here to guarantee that when we upgrade the Python driver, things will still work as expected and we won't leak any newly-added metadata without noticing.

Reply:
More thinking out loud here than anything... not sure if this is a fully-formed thought or not yet.

I'm wondering if this shouldn't be converted to a Python script, more specifically a Python test, to facilitate running on Jenkins.  At the moment we get a binary response from this script in the form of an exit code: either everything passed or something failed.  Integrating with the Python testing frameworks would allow us to provide responses for each distinct C* version.

Not sure it's essential... like I said above, more thinking out loud here.

Reply:
> Integrating with the Python testing frameworks would allow us to provide responses for each distinct C* version

That's a good point! I'll look into that, but maybe it could be a follow-up PR.

Reply:
Oh, absolutely.... this doesn't need to be done right now.  I'm :100: okay with handling this in a future ticket.

Reply:
I've created https://github.com/datastax/adelphi/issues/62 to track this future work